### PR TITLE
Upgraded to hedgehog 1.0

### PR DESCRIPTION
**Pull request by etorreborre**
As a quick follow-up to #28 here is a PR.

### CODE REVIEW

File: tasty-hedgehog.cabal [hunk 1]
```diff
@@ -24,7 +24,7 @@ library
   build-depends:       base >= 4.8 && <4.13
                      , tagged >= 0.8 && < 0.9
                      , tasty >= 0.11 && < 1.3
-                     , hedgehog >= 0.5 && < 0.7
+                     , hedgehog >= 1.0 && < 2.0
```
Comment (jacobstanley) [inline tasty-hedgehog.cabal]: I think these bounds are a bit extreme, you probably meant `>= 1.0 && < 1.1`

Comment (etorreborre) [reply tasty-hedgehog.cabal]: Right, this is fixed now.

**Comment by 23Skidoo** [comment]
Travis failure seems to be due to the latest `hedgehog` dropping support for GHC < 8.

**Comment by etorreborre** [comment]
@23Skidoo thanks, I removed the 7.x build now.

**Comment by LightAndLight** [comment]
Thanks @etorreborre @jacobstanley :)

**Pull request merged**